webapp/pages/index.vue

```diff
@@ -92,9 +92,39 @@ export default {
       ],
     }
   },
+
+  mounted() {
+    if (this.hashtag) {
+      this.changeFilterBubble({ tags_some: { name: this.hashtag } })
+    }
+    if (!this.hasAgreedToLatestTermsAndConditions) {
```

Comment:
Ideally, this should not happen client side when you mount the page component. Instead, as soon as the user object is loaded, you can redirect to another page immediately and ask for confirmation.